dev-qt/qttools/qttools-6.3.0.ebuild

```diff
@@ -0,0 +1,18 @@
+# Copyright 2021-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit qt6-build
+
+DESCRIPTION="Qt Tools"
+
+if [[ ${QT6_BUILD_TYPE} == release ]]; then
+	KEYWORDS="~amd64"
+fi
+
+DEPEND="
+	=dev-qt/qtbase-${PV}*
```

Comment:
several tools need at least the QtWidgets module but the dependency doesn't reflect that?

Reply:
Dependencies should be good now with the USE-flags added (may of course have missed some things).